tests/test_zones_json.py

```diff
@@ -7,6 +7,24 @@
 
 
 class ZonesJsonTestcase(unittest.TestCase):
+    # Add well-known sources that don't require config-based references here
+    GLOBAL_SOURCE_REFERENCES = {
```

Comment:
I think it'd be preferable to create a separate YAML reference sources file that contains these and any relevant metadata -- extracting a constant set here to hold them is something of a migration phase to figure out how many items exist and to spot patterns in them.